agent/containers/images/README.md

```diff
@@ -98,6 +98,10 @@ relation to what has been published already.  The push targets are:
 
  * `push-beta` - pushes each image by its `beta` tag
 
+There is also a special "push" target `publish` which pushes all the
+containers for each of the default distributions, but it pushes only the one
+tag defined by `IMAGE_TAG` and not the RPM version or Git hash tags.
```

Comment:
This will result in change that our users are going to notice.  Is there a particular reason why we don't want to push RPM and Git hash tags?

Reply:
As we discussed in the meeting, this mechanism is separate and distinct from the mechanism used for "releases", and, as such it has a different policy and approach.  In particular, the pushes are likely to be quite frequent, and we want to avoid a proliferation of images (and/or tags).  By pushing only a single tag, one which gets re-used, the clutter and resource consumption on the registry will be minimized, while still allowing users to access the "hot bits".

In our discussion, you expressed a requirement for being able to pull an image by its Git hash, so that a user can obtain a modicum of stability in the face of repeated updates to the branch or as a means for investigating the behavior of a particular change to the branch.  I support this request, but, until we have a mechanism for automatically deleting (or setting an expiration on) container images, I think that adding this now will create more problems than it solves, so I'd like to defer that until we have our management mechanism in place.

Reply:
Summary: this is NOT trying to be a release process, so this does not represent a change to the release process; git hash tags will be added once we have an expiration on images.